manifests/02-deployment.yaml

```diff
@@ -58,6 +60,8 @@ spec:
                   fieldPath: metadata.namespace
             - name: IMAGE
               value: openshift/origin-haproxy-router:v4.0
+            - name: CANARY_IMAGE
+              value: openshift/hello-openshift:latest
```

Comment:
Should we be specifying latest? It seems less deterministic.

Reply:
The CVO will override this value anyways, so I think `latest` is fine.

Reply:
For local testing, the latest tag is fine.